.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: 'node:14'
+      - image: 'node:18.10.0'
```

Comment:
Is there a reason we are pinning it to this particular version rather than just 18?

Reply:
@zenweasel 
Node version higher than `cimg/node:18.10.0` in example-storefront uses Ubuntu 22 which had compatibility issues with MongoDB. Hence we pinned at 18.10.0 in storefront and maintained the same node version here also